assets/src/apps/authoring/icons/icon-componentList.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
these raw svg assets should be stored in the "static" folder. since the authoring tool is closer to the "server side" we have access to the full hosting power of the server and should use it.